.config/lesskey

```diff
@@ -0,0 +1,11 @@
+# less automatically loads ~/.config/lesskey to configure itself, no need to set environment variable
+#command
+t  toggle-option --ignore-case\n
+# clear search highlighting with escape escape
+\e clear-search
+
+#env
+# set some sensible default options to always pass into invocations of less
+LESS = --mouse --wheel-lines=5 --quit-on-intr --ignore-case --RAW-CONTROL-CHARS --tabs=4 --window=-4--color=db$ --color=ug$ --color=sky$ --prompt=sInput= %f, Lines= %lt-%lb / %L$
```

Comment:
had to add --mouse to enable --wheel-lines
configuring the default short prompt, so no need to use long prompt anymore with --LONG-PROMPT